docs/pages/access-controls/login-rules.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: Login Rules (Preview)
+description: Transform User Traits with Login Rules
+layout: tocless-doc
+---
+
+<Admonition type="warning" title="Preview">
+  Login Rules are currently in Preview mode.
+</Admonition>
+
+When users log in to your Teleport cluster with a configured SSO provider,
+**Login Rules** can transform the traits provided by your IDP to meet your needs
+for configuring access within Teleport.
+
+Some use cases for Login Rules are:
+
+- When you need to modify a user trait based on logical rules, like
+  "users in group `db-admins` should also be added to group `db-users`",
+  Login Rules provide a powerful expression language to make these changes
+  without needing to modify claims in your IDP.
```

Comment:
Looks like our docs are inconsistent in their use of `IDP` vs `IdP`. Would be nice to fix this.

```
 teleport/docs on master [$!]  ❯
ag -c IdP
img/IBM/IBM_HA.svg:1
img/server-access/getting-started-diagram.svg:1
pages/access-controls/sso/oidc.mdx:1
pages/access-controls/sso.mdx:1
pages/includes/sso/idp-initiated.mdx:3
pages/connect-your-client/introduction.mdx:1
pages/server-access/guides/ssh-pam.mdx:1
pages/reference/cli.mdx:7
 teleport/docs on master [$!]  ➜
ag -c IDP
pages/access-controls/sso/okta.mdx:1
pages/access-controls/sso.mdx:1
pages/reference/cli.mdx:2
pages/reference/terraform-provider.mdx:1
```

Reply:
switched to IdP